BaseFilter/sections/specific.txt

```diff
@@ -349,6 +349,8 @@ olympics.com#$?##header-adv-banner { remove: true; }
 olympics.com#$#header.-sticky { top: 0 !important; }
 eurosport.com##.d3-top-advert
 eurosport.com##.d3-side-advert
+online2pdf.com##.c84a0a:has(> div.c84a10a.adbox_container)
+online2pdf.com##td:has(> .c84a1a > div.c84a4a.adbox_container)
```

Comment:
Usually you should fix it in `antiadblock.txt`, but I see there also ad leftovers.
Add this rule.
```suggestion
online2pdf.com##.adbox_container
```